identity-service/src/notifications/notificationQueue.js

```diff
@@ -37,44 +48,71 @@ async function addNotificationToBuffer (message, userId, tx, buffer, playSound,
   buffer.push(bufferObj)
 }
 
+async function _sendNotification (notifFunctionName, bufferObj) {
+  const logPrefix = `[notificationQueue:sendNotification] [${notifFunctionName}] [userId ${bufferObj.userId}]`
```

Comment:
I think we should try to keep the same log context as other things so this is easier to find "in - line"
https://github.com/AudiusProject/audius-protocol/blob/8218952343979a4d3dc87247b36bd0b1820d765e/identity-service/src/notifications/index.js#L371

Can we pass this through or at least copy?

Reply:
don't see any log context to pass through - this is just another generic logger, but gonna pass logger obj through